test/helpers/make-context.js

```diff
@@ -0,0 +1,22 @@
+var BufferList = require('bl');
+
+function makeContext(mod, path, options) {
```

Comment:
this generic `makeContext` is not usable for all tests that make a context. Some tests require a context from earlier in the lifecycle of the application. Should it be more descriptive in the export?